### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Aixiis**
Addition of the first "fr_fr.json" file !

### CODE REVIEW

File: src/main/resources/assets/sync/lang/fr_fr.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "itemGroup.sync.sync": "Sync",
+
+  "block.sync.shell_storage": "Stockage des capsules",
+  "block.sync.shell_constructor": "Fabricateur de capsules",
+  "block.sync.treadmill": "Tapis roulant",
+
+  "item.sync.sync_core": "Noyau de synchronisation",
+
+  "death.attack.sync.fingerstick": "%s est mort pour faire avancer la science !",
```
Comment (Kira-NT) [inline src/main/resources/assets/sync/lang/fr_fr.json:10]: Should an exclamation mark really stand one space apart from the text, or is it just a typo?

**Review by Kira-NT — CHANGES_REQUESTED** [review]
LGTM, but I have one question

File: src/main/resources/assets/sync/lang/fr_fr.json (hunk 1 above)
Comment (Aixiis) [reply src/main/resources/assets/sync/lang/fr_fr.json:10]: It is not required, it is just the way I write it, I (or you) can change it !

Comment (Kira-NT) [reply src/main/resources/assets/sync/lang/fr_fr.json:10]: > French. Yes, the whole language. In French, exclamation points, question marks, colons, and semi-colons (all forms of "high" punctuation) should always have a space preceding them. For example, if you wanted to text your friend about how much you hate being online, you'd say Je déteste être en ligne !

Oh, ok. It seems that I don't remember much from my french lessons :D

**Review by Kira-NT — APPROVED** [review]
Ok, everything looks nice

**Pull request merged**